CHANGELOG.md

```diff
@@ -275,6 +275,81 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - `@cumulus/message/Queue.getQueueName`
   - `@cumulus/message/Queue.getQueueNameByUrl`
 
+## v2.0.2+ Backport releases
```

Comment:
I guess this is helpful content, but what was the motivation for adding it? I would have been fine just labeling the other 2.0.x releases as "backports", which you did

Reply:
Regardless, since you already wrote it, might as well leave it in

Reply:
I think there's confusion over what the CHANGELOG on master is, given our backport structure.     

Just adding the [BACKPORT] label doesn't necessarily give readers context as to what that means (fixes mentioned aren't promised to have been brought into master/future versions as-described), and a disclaimer/gentle nudge in the right direction for the backport fixes feels warranted.